src/xrEngine/xr_input_xinput.cpp

```diff
@@ -522,6 +522,13 @@ keyboard_key* KeynameToPtr(pcstr name)
 void GetActionAllBinding(pcstr action, char* dst_buff, int dst_buff_sz)
 {
     int action_id = ActionNameToId(action);
+    if (action_id == kNOTBINDED)
+    {
+        if (dst_buff_sz > 0)
+            dst_buff[0] = '\0';
```

Comment:
There's no reason to write '\0' here, since it's a single char and not a string literal. Just put a plain 0 here.